app/validators/wpcc/salary_threshold_validator.rb

```diff
@@ -0,0 +1,22 @@
+module Wpcc
+  class SalaryThresholdValidator < ActiveModel::Validator
+    SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
+    SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
+    LOW_SALARY_THRESHOLDS = SALARY_FREQUENCIES.zip(SALARY_THRESHOLDS).to_h
```

Comment:
Yes, @margOnline IMHO we should be using the yaml files stored in the config and we need to add something on the readme that points to them (how to change rates, frequencies, etc).

@willhall88 perhaps they should be in the yaml file.